cranelift/interpreter/src/step.rs

```diff
@@ -349,8 +349,11 @@ where
     // Interpret a Cranelift instruction.
     Ok(match inst.opcode() {
         Opcode::Jump => {
-            let block = inst.branch_destination()[0];
-            continue_at(block)?
+            if let InstructionData::Jump { destination, .. } = inst {
+                continue_at(destination)?
+            } else {
+                unreachable!()
+            }
```

Comment:
This isn't necessary for the `br_table` changes, but it seemed nice to avoid the slice creation.